### DISCUSSION TITLE
第3节课笔记(18班)

### DISCUSSION THREAD

**Discussion by mattheliu** [InternLM/Tutorial#147] [笔记]
请提交笔记链接
Please submit the link to the notes.

**Comment by yestolife** [comment]
[基于 InternLM 和 LangChain 搭建你的知识库](http://t.csdnimg.cn/Iwdva)

**Comment by ggfengxuan** [comment]
[3.基于 InternLM 和 LangChain 搭建你的知识库笔记](https://v1y50l9n58a.feishu.cn/wiki/IoW2wYDBgidO0ikYwz9c9TW7n7g?from=from_copylink)

**Comment by ZMcc18** [comment]
https://www.yuque.com/zixul/avxnzn/igd0r6zboog1kpgk?singleDoc# 《lesson3 基于 InternLM 和 LangChain 搭建知识库 笔记》

**Comment by Heitao5200** [comment]
https://github.com/Heitao5200/my_blog/blob/master/docs/%E7%BB%84%E9%98%9F%E5%AD%A6%E4%B9%A0/%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%AE%9E%E6%88%98/TASK3.md

**Comment by scorpionfree98** [comment]
[书生 浦语大模型实战营-笔记+作业3](http://t.csdnimg.cn/dtwUm)

**Comment by DrZhang1208** [comment]
[基于 InternLM 和 LangChain 搭建知识库](https://blog.csdn.net/qq_42866033/article/details/135504587?spm=1001.2014.3001.5501)

**Comment by tackhwa** [comment]
[第3节课笔记+作业](https://nutritious-chill-7a9.notion.site/InternLM-LangChain-5fe5a09f8e3b4ea38d9dadf3223f5e8d?pvs=4)

**Comment by xwhclaire** [comment]
[Lecture 3 基于InternLM和LangChain搭建知识库](https://github.com/xwhclaire/StudyPackages/blob/main/LLM%20training/Lecture%203%20%E5%9F%BA%E4%BA%8EInternLM%E5%92%8CLangChain%E6%90%AD%E5%BB%BA%E4%B8%AA%E4%BA%BA%E7%9F%A5%E8%AF%86%E5%BA%93.md)

**Comment by ZiHAO-LI-cmd** [comment]
[书生·浦语大模型实战营 Lesson 3](http://t.csdnimg.cn/LdRNr)

**Comment by 2001wjh** [comment]
https://www.yuque.com/wjh2001/rdvfen/pdls73wr4a02ckf9?singleDoc# 《基于InternLM和LangChain搭建你的知识库（笔记&作业）》

**Comment by mattheliu** [comment]
https://github.com/mattheliu/gitblog/issues/3
